qa/tasks/cephfs/filesystem.py

```diff
@@ -515,7 +515,7 @@ def reach_max_mds(self):
                     count = len(list(self.get_ranks(status=status)))
             except:
                 # In Mimic, deactivation is done automatically:
-                log.debug("Error:\n{}".format(traceback.format_exc()))
+                log.info("Error:\n{}".format(traceback.format_exc()))
```

Comment:
could you prefix the title of your commit message with the subcomponent your are changing ? see https://github.com/ceph/ceph/blob/master/SubmittingPatches.rst#3-describe-your-changes

and "update" is far from informative, please summarize the change in the title, and put the rationales in the commit message.